portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/** Fetches the current date from the server and adds it to the page. */
```

Comment:
Let's update this comment since this method is not fetching the current date from the server. If that is what you intend to use the method for in the future, you can add a `TODO` comment describing that. The first paragraph of this intellij doc decribes when and why we might make `TODO` comments: https://www.jetbrains.com/help/idea/using-todo.html